engine/link/mystery_gift.asm

```diff
@@ -33,18 +33,25 @@ DoMysteryGift:
 	ld de, .String_PressAToLink_BToCancel
 	call PlaceString
 	call WaitBGMap
-
-	; Prepare the first of two messages for wMysteryGiftPartnerData
 	farcall StageDataForMysteryGift
 	call ClearMysteryGiftTrainer
+	vc_hook infrared_fake_0
+if DEF(_CRYSTALVC)
+	; Prepare the first of two messages for wMysteryGiftPartnerData
```

Comment:
Maybe this comment was here in the old version of pokecrystal, but please move it back.

Reply:
https://github.com/pret/pokecrystal/pull/882/commits/4059f57948a750d1b1262152136a9be1e53b4969 should resolve this.